tensorflow/python/data/util/random_seed_test.py

```diff
@@ -18,64 +18,120 @@
 from __future__ import division
 from __future__ import print_function
 
+import functools
+
+from absl.testing import parameterized
+
+from tensorflow.python.data.kernel_tests import test_base
 from tensorflow.python.data.util import random_seed as data_random_seed
-from tensorflow.python.eager import context
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import constant_op
 from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import random_seed
-from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 
 
-class RandomSeedTest(test.TestCase):
+# NOTE(vikoth18): Arguments of parameterized tests are lifted into lambdas to make
+# sure they are not executed before the (eager- or graph-mode) test environment
+# has been set up.
+
+def _test_random_seed_combinations():
+
+  cases = [
+      # Each test case is a tuple with input to get_seed:
+      # (input_graph_seed, input_op_seed)
+      # and output from get_seed:
+      # (output_graph_seed, output_op_seed)
+      (
+          "CASE_0",
```

Comment:
replace "CASE" with "test_case" throughout this PR

Reply:
Replaced "CASE" with "TestCase" to be consistent with the style of names of NamedObjects in other tests. Let me know if it's ok.